Jenkinsfile

```diff
@@ -32,7 +32,7 @@ node('ubuntu-zion') {
       OsTools.runSafe(this, "docker system prune -a -f")
 
       checkout scm
-      branch = scm.branches[0].name
+      branch = env.BRANCH_NAME
```

Comment:
This evaluates to null for me.

Please see: https://jenkins.zion.aws.s/job/integrations/job/jcava-docker-nexus3/1/console

[Pipeline] echo
branch = null

Reply:
Thanks, looks like that's only set in multibranch pipeline jobs. 
11cd3c3